src/expire-tiles.cpp

```diff
@@ -122,6 +122,21 @@ int expire_tiles::normalise_tile_x_coord(int x)
     return x;
 }
 
+/**
+ * Converts from target coordinates to tile coordinates.
+ *
+ * The zoom level for the coordinates is explicitly given in the
+ * parameter map_width.
+ */
+static void coords_to_tile(reprojection const &projection, double *tilex,
+                           double *tiley, double lon, double lat, int map_width)
```

Comment:
If you make that a member of expire_tiles, you can save the projection and map_width parameter. And while you are at it, I would suggest to move output parameters to the end of the parameter list.

Reply:
I added a commit with this.